cobc/parser.y

```diff
@@ -5318,6 +5338,15 @@ select_clause:
 */
 ;
 
+with_clause:
```

Comment:
Shouldn't that do anything? Here's parsing only which means it possibly needs a PENDING above if that clause is enabled.